speccy/machine.go

```diff
@@ -49,7 +48,15 @@ func (m *Machine) Start() error {
 	}
 	m.z.RegisterInputHandler(func(addr uint16) byte { return m.keys.keyboardInputHandler(addr) })
 	every := time.Second / 50
+
 	go func() {
+		runtime.LockOSThread()
+		sdl.Init(sdl.INIT_EVERYTHING)
+		screen, err := NewScreen(m.z.Mem)
```

Comment:
No need for a screen local var, can init the struct member directly.

Reply:
Merged NewScreen call with struct member init